Update for Cake 4 Changes

JacobAGTyler opened the pull request.

JacobAGTyler committed:
Update for Cake 4 Changes

JacobAGTyler committed:
Cake 4 Updates

josbeir commented:
Hi, tests are failing, looks like some changes are necessary to the testing bootstrap. It would be great if you have the time to check that out.

dereuromark commented on composer.json:
```diff
@@ -16,7 +16,7 @@
     "require": {
         "php": ">=7.1",
-        "cakephp/cakephp": "^3.6",
+        "cakephp/cakephp": "^3.6|4.x-dev",
```
You should never do this. Pretty sure 3.x and 4.x cannot work both here.

JacobAGTyler replied:
I've updated it to dev-master for the 4.0.1 fix #14033 related to issue #14032. It can be 4.0.1 after CakePHP/CakePHP 4.0.1 release.

JacobAGTyler committed:
Cake 4 Updates

Cake Dev-Master can be reverted to this version after 4.0.1 release which includes fix: #14033 on issue #14032.

JacobAGTyler committed:
Merge branch 'patch-1' into master

JacobAGTyler committed:
Merge pull request #1 from JacobAGTyler/master

Test Failure Fix

JacobAGTyler committed:
Travis Update

JacobAGTyler committed:
Merge remote-tracking branch 'origin/master'

JacobAGTyler committed:
Merge branch 'master' into patch-1

JacobAGTyler committed:
Travis Update

JacobAGTyler committed:
Remove debug.

JacobAGTyler commented:
Think this is good to go, have resolved all the issues I can see.

josbeir commented:
thanks! great work @JacobAGTyler !

saeideng commented on composer.json:
```diff
@@ -16,11 +16,11 @@
-        "league/flysystem": "^1.0"
+        "cakephp/cakephp": "dev-master",
+        "league/flysystem": "^1.0",
+        "phpunit/phpunit": "^8.0"
```
ideally `"phpunit/phpunit` should be in `require-dev` section

josbeir replied:
I fixed that in a newer commit, was faster than doing a review and waiting until the stuff is in there :)

JacobAGTyler replied:
Awesome thanks

josbeir commented:
I fixed that in a newer commit, was faster than doing a review and waiting until the stuff is in there :)

On Wed, Dec 25, 2019, at 14:23, saeideng wrote:
> *@saeideng* commented on this pull request.

> 
> In composer.json <https://github.com/josbeir/cakephp-filesystem/pull/1#discussion_r361312142>:

> > @@ -16,11 +16,11 @@
     },
     "require": {
         "php": ">=7.1",
-        "cakephp/cakephp": "^3.6",
-        "league/flysystem": "^1.0"
+        "cakephp/cakephp": "dev-master",
+        "league/flysystem": "^1.0",
+        "phpunit/phpunit": "^8.0"
> 
> ideally `"phpunit/phpunit` should be in `require-dev` section

> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub <https://github.com/josbeir/cakephp-filesystem/pull/1?email_source=notifications&email_token=AAAGLSSOSGHBQIN5CO5A6PDQ2NNEZA5CNFSM4J3DKCSKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCQGIAXQ#pullrequestreview-336363614>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAAGLSR7PHR2SZ772ETK7IDQ2NNEZANCNFSM4J3DKCSA>.
